CONTRIBUTING.md

```diff
@@ -5,29 +5,37 @@
 
 ## Getting Started
 
-To work on this project please create an account on Cloudflare and Discord.
-Also, create a personal discord server.
-Copy the guild id of your server and save it in a .env file as `DISCORD_GUILD_ID`
-
-To get the server ID for the first parameter, open Discord, go to Settings > Advanced and enable developer mode.
-Then, right-click on the server title and select "Copy ID" to get the guild ID.
-
+To contribute to this project, you have to set up few things first.
+
+- Create an account on Cloudfare Workers and Discord if you don't already have one
+- Create a personal discord server
+- Fork and clone this repo
```

Comment:
this is optional, If you are a member of the community you don't have to do this.

Reply:
you still have to clone right?

Reply:
yes